package.json

```diff
@@ -89,7 +89,8 @@
     "superscript-text": "^1.0.0",
     "tinycolor2": "^1.3.0",
     "topojson-client": "^2.1.0",
-    "webgl-context": "^2.2.0"
+    "webgl-context": "^2.2.0",
+    "world-calendars": "0.1.0"
```

Comment:
👍  when we get @n-riesco 's Chinese in `world-calendars` I'll change the pattern here - and probably bump it to 1.0.0 at that point since it's doing its job.